desktop-ui/desktop-ui.cpp

```diff
@@ -144,6 +163,17 @@ auto nall::main(Arguments arguments) -> void {
     if(file::exists(argument)) program.startGameLoad.append(argument);
   }
 
+#if defined(PLATFORM_WINDOWS)	
+ /* Force and Change Settings By Commandline */
+  if ( SettingsGeneralsBorderless > 0)
+  {
+	  if ( SettingsGeneralsBorderless == 1)
```

Comment:
nit: single line if statements should have the body on the same line, if it's too large to fit horizontally, use braces.